src/test/java/lotto/LottoTotalNumberUtilsTest.java

```diff
@@ -16,29 +17,28 @@
     void 로또_전체_번호_크기() {
         // given
         Collection<Integer> lottoTotalNumbers = LottoTotalNumberUtils.getLottoTotalNumbers();
-        
+
         // when
         int size = lottoTotalNumbers.size();
 
         // then
         assertThat(size).isEqualTo(45);
     }
 
-    // 1 ~ 45가 전부 유효한지만 체크하는 메서드 없나? Assertj 문서를 못보겠다. ㅜ.ㅜ
-//    @DisplayName(value = "로또를 선택하기 위한 1 ~ 45의 숫자가 모두 있는지 확인")
-//    @Test
-//    void 로또_전체_번호_체크() {
-//        // given
-//        List<Integer> lottoTotalNumbers = LottoTotalNumberUtils.getLottoTotalNumbers();
-//        System.out.println("lottoTotalNumbers = " + lottoTotalNumbers.size());
-//
-//        // when
-//        List<Integer> expected = new ArrayList<>();
-//        for (int i = 1; i <= 44 ; i++) {
-//            expected.add(i);
-//        }
-//
-//        // then
-//        assertThat(lottoTotalNumbers).;
-//    }
+    @DisplayName(value = "로또를 선택하기 위한 1 ~ 45의 숫자가 모두 있는지 확인")
+    @Test
+    void 로또_전체_번호_체크() {
+        // given
+        List<Integer> lottoTotalNumbers = LottoTotalNumberUtils.getLottoTotalNumbers();
+        Collections.sort(lottoTotalNumbers);
+
+        // when
+        List<Integer> expected = new ArrayList<>();
+        for (int i = 1; i <= 45; i++) {
```

Comment:
```suggestion
        for (int i = LottoTotalNumberUtils.LOTTO_NUMBER_MINIMUM; i <= LottoTotalNumberUtils.LOTTO_NUMBER_MAXIMUM; i++) {
```
이미 정의된 상수를 이용해보면 어떨까요? 😄